win32/devdoc/threadpool_win32.md

```diff
@@ -74,19 +74,13 @@ static void threadpool_dispose(THREADPOOL* threadpool)
 
 `threadpool_dispose` frees all resources associated with threadpool.
 
-**SRS_THREADPOOL_WIN32_01_005: [** `threadpool_dispose` shall free all resources associated with `threadpool`. **]**
-
-**S_R_S_THREADPOOL_WIN32_01_016: [** If `threadpool` is `NULL`, `threadpool_dispose` shall return. **]**
-
-**SRS_THREADPOOL_WIN32_01_006: [** While `threadpool` is OPENING or CLOSING, `threadpool_dispose` shall wait for the open to complete either successfully or with error. **]**
-
 **SRS_THREADPOOL_WIN32_01_007: [** `threadpool_dispose` shall perform an implicit close. **]**
```

Comment:
This spec is obsolete now that close/open are not used any longer, so the spec should be removed. From specs/tests and code please.

Reply:
Okay, it said implicit close and the next 3 lines are subsection of this section. The close is happening but not in threadpool_close.

Reply:
The change is to not have an open/close state any longer. In other words to help understand my thought process: what does "close" mean if we do not have an open/closed state for the module?

Reply:
We have `CloseThreadpoolCleanupGroupMembers` and `CloseThreadpoolCleanupGroup` which essentially means we are closing the threadpool related items in this section

Reply:
Does the threadpool have an open state vs a closed state any longer? I don't think it does. If it would have a closed/opened state it would be modeled in a state variable (either owned by the threadpool object or delegated to an SM object).

The spec to perform an implicit close in the destroy is written for objects that have an open/closed state.

HTH.

Reply:
Removed the specs